contrib/cirrus/lib.sh

```diff
@@ -164,8 +164,7 @@ in_podman() {
 
     msg "Gathering env. vars. to pass-through into container."
     for envname in $(awk 'BEGIN{for(v in ENVIRON) print v}' | sort | \
-                     egrep "$envrx" | egrep -v "$SECRET_ENV_RE" | \
-                     egrep -v "^CIRRUS_.+(MESSAGE|TITLE)")
+                     egrep "$envrx" | egrep -v "$SECRET_ENV_RE");
```

Comment:
This removes the `CIRRUS_MESSAGE` filtering. I tried to follow the commit history, to understand why that was believed necessary, but can't figure it out. Do you have any recollection of why that was added, and whether it is your express intention to remove it here?

Reply:
It's a bad design, predating the refactoring work you did in podman.  It was ignored previously out of caution - it get's passed into the containers via an `--env-file` which doesn't handle multi-line strings (or the raw-string bash syntax `$'<stuff>'`).  However in this case we don't care.  We only care if `[CI:NEXT]` is present or not.

Reply:
Added a `TODO`.